examples/sslecho/test.d

```diff
@@ -0,0 +1,25 @@
+/// Runs the server and the client in a fork for CI
+module test;
+
+import core.thread;
+import core.time;
+
+import std.process;
+import std.stdio;
+
+int main ()
+{
+    auto serverPid = spawnProcess([ "./sslecho", "s" ]);
+    writeln("Server has been spawned");
+    Thread.sleep(3.seconds);
```

Comment:
The `sleep` can be replaced with making `sslecho` print something like `"Ready\n"`, using `pipeProcess` here and doing a `p.stdout.readln`.